src/components/Home.js

```diff
@@ -1,29 +1,12 @@
-import React, { useState } from 'react';
+import React from 'react';
 import Books from './ListBooks';
 import AddBook from './AddBook';
 
 function Home() {
-  const [books] = useState([
-    {
-      id: 1,
-      title: 'the hunger games',
-      author: 'suzanne collins',
-    },
-    {
-      id: 2,
-      title: 'Dune',
-      author: 'Frank Herbert',
-    },
-    {
-      id: 3,
-      title: 'capital in the twenty-first century',
-      author: 'suzanne collins',
-    },
-  ]);
   return (
     <>
       <div>
-        <Books books={books} />
+        <Books />
         <AddBook />
```

Comment:
- [ ] I'm not able to add a new book due to state component issues. Kindly view the error log below to fix this and have it work properly. Welldone 👏 

### Error view:
![image](https://github.com/mariegrace31/bookstore/assets/50990925/c73eac05-937f-4112-a551-afad2051ae10)